kernel/src/schema/derive_macro_utils.rs

```diff
@@ -7,6 +7,11 @@ use crate::schema::{ArrayType, DataType, MapType, StructField, ToSchema};
 
 /// Converts a type to a [`DataType`]. Implemented for the primitive types and automatically derived
 /// for all types that implement [`ToSchema`].
+#[cfg(feature = "internal-api")]
+pub trait ToDataType {
+    fn to_data_type() -> DataType;
+}
+#[cfg(not(feature = "internal-api"))]
```

Comment:
does directly doing `#[internal_api]` not work?

Reply:
sorry should have been more specific - you can see many other instances of us marking things as `#[internal_api]` which is a decorator that will be less verbose than the manual `cfg(...)` - I _think_ it should work here, but not on the module declaration

Reply:
Ahh, got it, will review

Reply:
Alright, it worked nicely, changed